deps/rabbitmq_shovel_management/src/rabbit_shovel_mgmt.erl

```diff
@@ -107,52 +99,28 @@ filter_vhost_req(List, ReqData) ->
                            pget(vhost, I) =:= VHost]
     end.
 
-%% Allow users to see things in the vhosts they are authorised. But
-%% static shovels do not have a vhost, so only allow admins (not
-%% monitors) to see them.
-filter_vhost_user(List, _ReqData, #context{user = User = #user{tags = Tags}}) ->
-    VHosts = rabbit_mgmt_util:list_login_vhosts_names(User, undefined),
-    [I || I <- List, case pget(vhost, I) of
-                         undefined -> lists:member(administrator, Tags);
-                         VHost     -> lists:member(VHost, VHosts)
-                     end].
-
-status(ReqData, Context) ->
-    filter_vhost_user(
-      lists:append([status(Node) || Node <- [node() | nodes()]]),
-      ReqData, Context).
-
-status(Node) ->
-    case rpc:call(Node, rabbit_shovel_status, status, [], infinity) of
-        {badrpc, {'EXIT', _}} ->
-            [];
-        Status ->
-            [format(Node, I) || I <- Status]
+restart_shovel(VHost, Name, Node) ->
+    rabbit_log:info("Calling rabbit_shovel_util:restart_shovel on Node '~s' for shovel '~s' in vhost '~s'", [Node, Name, VHost]),
+    case rpc:call(Node, rabbit_shovel_util, restart_shovel, [VHost, Name], infinity) of
```

Comment:
`infinity` timeouts is a very bad idea. We have years of experience to back this claim.